src/pages/ReimbursementAccount/RequestorOnfidoStep.js

```diff
@@ -31,6 +31,16 @@ class RequestorOnfidoStep extends React.Component {
     constructor(props) {
         super(props);
         this.submit = this.submit.bind(this);
+        this.goBack = this.goBack.bind(this);
+    }
+
+    goBack() {
+        const onfidoBack = document.querySelector('.onfido-sdk-ui-NavigationBar-back');
```

Comment:
This caused deploy blocker - https://github.com/Expensify/App/issues/24343.
`document` doesn't exist on native.

Reply:
Dang, I didn't encountered this during my testing on offline...
@vipin87 Can you rise the follow-up for the issue with proper platform handling?